test/fakeIngestion/standalone/src/main/java/com/microsoft/applicationinsights/test/fakeingestion/MockedAppInsightsIngestionServer.java

```diff
@@ -93,6 +93,19 @@ public int getCountForType(String type) {
         return dataItems;
     }
 
+    public <T extends Domain> List<T> getAllMessageData() {
```

Comment:
can you use getMessageDataInRequest?

Reply:
getMessageDataInRequest will filter logs based on ai.operation.id

Reply:
can you using `getMessageDataInRequest` now that you've moved the logging into the controller?